bin/metrics-vertx.rb

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env ruby
+#
+# metrics-vertx
+#
+# DESCRIPTION:
+#  metrics-vertx get metrics from VertX
+#
+# OUTPUT:
+#   metric-data
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: rest-clien
+#
+# USAGE:
+#
+#
+# NOTES:
+#
+# LICENSE:
+#   Zubov Yuri <yury.zubau@gmail.com> sponsored by Actility, https://www.actility.com
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/metric/cli'
+require 'rest-client'
+require 'json'
+
+class MetricsVertX < Sensu::Plugin::Metric::CLI::Graphite
+  option :endpoint,
+         short: '-p ENDPOINT',
+         long: '--endpointn ENDPOINT',
+         description: 'VertX Endpoint',
+         default: 'http://localhost:8080/rest/metrics'
+
+  option :scheme,
+         description: 'Metric naming scheme, text to prepend to metric',
+         short: '-S SCHEME',
+         long: '--scheme SCHEME',
+         default: "#{Socket.gethostname}.vertx"
+
+  def request
```

Comment:
as we have the same logic can we break this out into a lib to avoid duplication?

Reply:
@majormoses move to common file.